pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[project]
+name = "streamlit-dataframe-summary"
+version = "0.1.0"
+description = "Streamlit component to summarize a DataFrame"
+readme = "README.md"
+requires-python = ">=3.12"
+authors = [{ name = "Arnaud Miribel", email = "arnaudmiribel@gmail.com" }]
+dependencies = ["plotly>=5.24.1", "seaborn>=0.13.2", "streamlit>=1.38"]
```

Comment:
I'm fine with having `streamlit>=1.38` for now, especially since I'm using un-experimental commands for st.dialog and st.fragment - but that could easily be supported for earlier versions though.

Reply:
For what it's worth, I have uploaded a separate version for our SiS apps that uses experimental commands, and also doesn't use Material icons